assets/css/app.css

```diff
@@ -2,6 +2,12 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
+[data-phx-main] {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
```

Comment:
This, along with some styles on the body helps make the minimum height of our app the full screen so that the footer is pushed all the way to the bottom without content being in between.